cli/azd/internal/appdetect/appdetect_test.go

```diff
@@ -173,32 +173,6 @@ func TestDetect(t *testing.T) {
 						JavaProjectOptionWinMavenWrapperPath:   filepath.Join(dir, "java", "mvnw.cmd"),
 					},
 				},
-				{
-					Language:      Java,
-					Path:          "java-multi-levels/submodule/subsubmodule1",
-					DetectionRule: "Inferred by presence of: pom.xml",
-					Metadata: Metadata{
-						ApplicationName: "subsubmodule1",
-					},
-					Options: map[string]interface{}{
-						JavaProjectOptionParentPomDir:          filepath.Join(dir, "java-multi-levels"),
-						JavaProjectOptionPosixMavenWrapperPath: filepath.Join(dir, "java-multi-levels", "mvnw"),
-						JavaProjectOptionWinMavenWrapperPath:   filepath.Join(dir, "java-multi-levels", "mvnw.cmd"),
-					},
-				},
-				{
-					Language:      Java,
-					Path:          "java-multi-levels/submodule/subsubmodule2",
-					DetectionRule: "Inferred by presence of: pom.xml",
-					Metadata: Metadata{
-						ApplicationName: "subsubmodule2",
-					},
-					Options: map[string]interface{}{
-						JavaProjectOptionParentPomDir:          filepath.Join(dir, "java-multi-levels"),
-						JavaProjectOptionPosixMavenWrapperPath: filepath.Join(dir, "java-multi-levels", "mvnw"),
-						JavaProjectOptionWinMavenWrapperPath:   filepath.Join(dir, "java-multi-levels", "mvnw.cmd"),
-					},
-				},
```

Comment:
Why deleting these tests?

Reply:
I originally intend to delete these redundant test cases to fasten UT speed.